package.json

```diff
@@ -9,8 +9,12 @@
     "lint": "next lint"
   },
```

Comment:
can we pin node/yarn versions via volta? shall we try yarn 3?

Reply:
Yes, I can make this change